examples/datasets/openai_prm800k.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 The HuggingFace Inc. team. All rights reserved.
```

Comment:
You can remove this file in favour of https://github.com/huggingface/trl/blob/main/examples/datasets/prm800k.py